gaffer-as-a-service/gaas-rest/pom.xml

```diff
@@ -159,6 +159,11 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-text</artifactId>
         </dependency>
+        <dependency>
```

Comment:
There seems to be unresolved merge conflicts here and elsewhere in the branch

Reply:
@t92549  resolved merge conflicts.